docs/google-analytics.md

```diff
@@ -0,0 +1,78 @@
+## how google analytics works
+
+Since angular is a single page application, I implemented router checking. So when a user activates the Router function, it is recorded as visiting a new part of the page.
+
+For optimal user tracking and continued work on Treecreate.dk after we have launched, it is important that we work eventEmitters into the more important actions of our users. ex:
+
+1. how long they spend creating a design
+2. how many designs they create
+3. how go from one page to the other
+   etc etc.
```

Comment:
In markdown files, a newline in the raw text does not correspond directly to a newline in a formatted file
Because of this your list actually looks like this:
```
...
3. how go from one page to the other etc etc.
```
If you wanted the `etc etc` to be on a new line, you can add a `\` character at the end of the previous line

Reply:
cool, thanks

Reply:
ill remember that